lib/spack/spack/database.py

```diff
@@ -835,7 +922,9 @@ def query_one(self, query_spec, known=any, installed=True):
         query. Returns None if no installed package matches.
 
         """
-        concrete_specs = self.query(query_spec, known, installed)
+        concrete_specs = self.query(
+            query_spec, known=known, installed=installed
+        )
```

Comment:
minor nit: can we put this on the prior line?  I wish there was a way to make it more consistent via the flake8 check but I don't think there is.

Reply:
You mean like:
```python
concrete_specs = self.query(
    query_spec, known=known, installed=installed)
```
?

p.s. we should definitely check [yapf](https://github.com/google/yapf) once again and consider using it like `go fmt`